src/main/java/org/intellij/sdk/codesync/ui/dialogs/PricingAlertDialog.java

```diff
@@ -27,10 +28,27 @@ public PricingAlertDialog(Boolean isOrgRepo, Boolean canAvailTrial, String prici
         this.pricingURL = pricingURL;
 
         if (canAvailTrial) {
-            upgradeButtonText = "Try Pro for free";
+            upgradeButtonText = isOrgRepo ? NotificationButton.TRY_TEAM_FOR_FREE: NotificationButton.TRY_PRO_FOR_FREE;
             secondaryMessage = isOrgRepo ? Notification.TRY_ORG_PRO_FOR_FREE: Notification.TRY_PRO_FOR_FREE;
```

Comment:
Not part of this PR but we can rename TRY_ORG_PRO_FOR_FREE to TRY_TEAM_PLAN_FOR_FREE